tests/_test_mscolab/test_server.py

```diff
@@ -139,6 +142,19 @@ def test_delete_user(self):
             assert response.status_code == 200
             assert response.data.decode('utf-8') == "False"
 
+            # create user again, this time the user has a profile image set
+            assert add_user(self.userdata[0], self.userdata[1], self.userdata[2])
+            token = self._get_token(test_client, self.userdata)
+            response = self._upload_profile_image(test_client, token, self.userdata[0])
+            assert response.status_code == 200  # this will ensure image was uploaded
+            response = test_client.post('/delete_own_account', data={"token": token})
+            assert response.status_code == 200
+            data = json.loads(response.data.decode('utf-8'))
+            assert data["success"] is True
+            response = test_client.post('/delete_own_account', data={"token": "dsdsds"})
```

Comment:
what should this verify regarding to delete the own user account? The account was deleted. The token used is wrong.